spec.html

```diff
@@ -18299,29 +18300,29 @@ <h1>Runtime Semantics: Evaluation</h1>
         <h1>
           Runtime Semantics: PropertyDefinitionEvaluation (
             _object_: unknown,
-          ): a Completion Record normally containing a Boolean
+          ): a Completion Record normally containing ~unused~
         </h1>
         <dl class="header">
         </dl>
         <emu-grammar>PropertyDefinitionList : PropertyDefinitionList `,` PropertyDefinition</emu-grammar>
         <emu-alg>
           1. Perform ? PropertyDefinitionEvaluation of |PropertyDefinitionList| with argument _object_.
-          1. Return ? PropertyDefinitionEvaluation of |PropertyDefinition| with argument _object_.
+          1. Perform ? PropertyDefinitionEvaluation of |PropertyDefinition| with argument _object_.
```

Comment:
These still have to return `~unused~`. Algorithms cannot reach the end without hitting a return. That would be an editorial error.

Reply:
> Algorithms cannot reach the end without hitting a return. That would be an editorial error.

Not currently, and nothing as of this PR technically says that, I think? I agree it's best avoided at least until we [give a definition](https://github.com/tc39/ecma262/pull/2397), but I don't think it's clearly an editorial error for an AO not to `return` as long as the AO is only invoked with `Perform`.

Reply:
If it's not defined, it's not permitted IMO. Anyway, we shouldn't do it here or anywhere else for now.

Reply:
I've been meaning to explicitly document at least the early exit behaviour of "return" in the algorithm conventions section.